sql/hive/src/main/scala/org/apache/spark/sql/hive/HiveSessionCatalog.scala

```diff
@@ -75,6 +77,33 @@ private[sql] class HiveSessionCatalog(
   // | Methods and fields for interacting with HiveMetastoreCatalog |
   // ----------------------------------------------------------------
 
+  override def validateName(name: String): Boolean = {
+    super.validateName(name)
```

Comment:
`super.validateName(Name)`? I don't think we need this - we are not using its results anyway.